### DISCUSSION TITLE
Packages/com.unity.localization/manual/EditModeSupport

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#624] [Notes]
# Packages/com.unity.localization/manual/EditModeSupport

https://docs.unity3d.com/Packages/com.unity.localization@latest/index.html?subfolder=%2Fmanual%2FEditModeSupport.html

**Comment by Cyantoo** [comment]
The Edit Mode support also allows you to use LocalizationSettings.StringDatabase.GetTable() in an Editor Script. You need to open the Localization Scene Controls window and choose a locale for it to work properly.
